com.unity.render-pipelines.universal/Editor/2D/Renderer2DDataEditor.cs

```diff
@@ -25,6 +25,7 @@ class Styles
             public static readonly GUIContent blendFactorAdditive = EditorGUIUtility.TrTextContent("Additive");
             public static readonly GUIContent useDepthStencilBuffer = EditorGUIUtility.TrTextContent("Use Depth/Stencil Buffer", "Uncheck this when you are certain you don't use any feature that requires the depth/stencil buffer (e.g. Sprite Mask). Not using the depth/stencil buffer may improve performance, especially on mobile platforms.");
             public static readonly GUIContent postProcessData = EditorGUIUtility.TrTextContent("Post-processing Data", "Resources (textures, shaders, etc.) required by post-processing effects.");
+            public static readonly GUIContent enableBatching = EditorGUIUtility.TrTextContent("Enable Batching", "Enable Batching for Shape / Parametric Lights.");
```

Comment:
Parametric light don't exist anymore

Reply:
Seems it will still be supported based on discussion with @unity-cchu and @rainsing